src/routes/RootRouter/RootRouter.js

```diff
@@ -41,51 +41,23 @@ export default function RootRouter () {
       <Route component={AuthLayoutRouter} />
     )
   }
-
   if (!isAuthorized) {
-    // TODO: Can NonAuthLayoutRouter and PublicLayoutRouter merge?
     return (
       <Switch>
-        <Route path='/post/:id' component={PublicLayoutRouter} />
-        <Route path='/public/groups' exact component={NonAuthLayoutRouter} />
-        <Route path='/public' component={PublicLayoutRouter} />
-        <Route path={'(.*)' + POST_DETAIL_MATCH} component={CheckPublicPost} />
+        <Route
+          path='/:context(groups)/:groupSlug/join/:accessCode'
+          component={NonAuthLayoutRouter}
+        />
+        <Route
+          path={[
+            '/:context(public)/:view(map|groups)?', // so I will need to add something here to great public, non-auth group page up? and without stopping the group join mechanism detailed by Loren
```

Comment:
I don't understand-- what are you saying that I'm saying that I think we need to do 😂 ?  In general to remember/confirm what should happen here I'd test the couple routes for signing-up (invite link) or token-auth'ing (email invite) to a join a group when not-authorized. If those flows work, we're good, if not we may need to adjust something here. On glance without digging into it now, I suspect this is all fine as is, but test it out if you're not sure.

Reply:
redundant comment, sorry